README.md

```diff
@@ -6,16 +6,22 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/c3ba87d04fe24b8f881252705e51cc29)](https://www.codacy.com/app/CFER/serverless-vpc-discovery?utm_source=github.com&utm_medium=referral&utm_content=amplify-education/serverless-vpc-discovery&utm_campaign=badger)
 [![npm downloads](https://img.shields.io/npm/dt/serverless-vpc-discovery.svg?style=flat)](https://www.npmjs.com/package/serverless-vpc-discovery)
 
-The vpc discovery plugin takes the given vpc, subnet, and security group names in the serverless file to setup the vpc configuration for the lambda.
+The vpc discovery plugin takes the given vpc name, subnet tag key/value, and security group tag key/value or names in the serverless file to setup the vpc configuration for the lambda.
 
 Basically we use this config:
 ```
 vpcDiscovery:
-    vpcName: '${opt:env}'
-    subnetNames: # optional if securityGroupNames are specified
-      - '${opt:env}_<name of subnet>'
-    securityGroupNames: # optional if subnetNames are specified
-      - '${opt:env}_<name of security group>'
+    vpcName: '<vpc_name>'
+    subnets:
+      - tagKey: <tag_name>
+        tagValues:
+          - '<tag_vale>'
+    securityGroups:
+      - tagKey: <tag_name>
+        tagValues:
+          - '<tag_value>'
+      - names:
```

Comment:
Nitpick: Maybe remove this so people don't think `names` is required if using `tagKey`.